### PR TITLE
Add aura zapper and claiming rewards scripts

### PR DESCRIPTION
**Pull request by petrovska-petro**
Tackles #626 and #627

```
brownie run issue/626/claim_rewards_and_unlock_aura

brownie run badger/claim_aura_rewards
```

### CODE REVIEW

File: scripts/badger/claim_aura_rewards.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main(
+    msig_name="treasury_voter_multisig",
+    redirect_rewards=False,
+    destination="0xA9ed98B5Fb8428d68664f3C5027c62A10d45826b",  # defaults to voter_msig
+):
```
Comment (gosuto-inzasheru) [inline scripts/badger/claim_aura_rewards.py:9]: this is super confusing, what is the purpose of this script?

**Review by gosuto-inzasheru — COMMENTED** [review]
i think the option stuff can be done a bit more intuitive, will make a pr

File: great_ape_safe/ape_api/aura.py [hunk 2]
```diff
@@ -1,28 +1,66 @@
...
+            self.zap.claimRewards([], [], [], [], 0, 0, 0, option)
+
+            # checkin five tokens, doubt will distribute ever more than that
+            for i in range(5):
+                try:
+                    reward_token = self.aura_locker.rewardTokens(i)
+
+                    assert self.safe.contract(reward_token).balanceOf(self.safe) > 0
+                except VirtualMachineError:
+                    return
```
[the remark this answers is not in the document]
Comment (petrovska-petro) [reply great_ape_safe/ape_api/aura.py:61]: they serve the same purpose, just that the reading for the rewards tokens is done for each in different way, so we need an exception catch for those ranges out of the index

File: scripts/badger/claim_aura_rewards.py (hunk 1 above)
Comment (petrovska-petro) [reply scripts/badger/claim_aura_rewards.py:9]: it is a generalised script for claiming rewards from aura. allows claiming from different msigs and as usual, if we need to redirect rewards to voter, we can do it in one go within the same script

File: scripts/issue/627/graviaura_deposit.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    graviaura = safe.contract(r.sett_vaults.graviAURA)
+
+    safe.take_snapshot(
+        tokens=[aura, graviaura]
+    )
+
+    aura.approve(graviaura, aura.balanceOf(safe))
+    graviaura.depositAll()
+
+    safe.post_safe_tx()
```
[the remark this answers is not in the document]
Comment (petrovska-petro) [reply scripts/issue/627/graviaura_deposit.py:16]: while tackling the meta issue i decided to batch a couple of issues which were pretty much interconnected/similar context, that's why the branch is not narrow down to #627 only

[the remark this answers is not in the document]
Comment (petrovska-petro) [reply scripts/issue/627/graviaura_deposit.py:16]: also, this script was meant to do the claiming action of the penalised rewards available for voter, but was front-runned by Trit

**Review by gosuto-inzasheru — APPROVED** [review]
awesome!
- adds claim functions to the aura class (#627)
- general script to claim all from aura booster from any multisig
- specific script to unlock locked aura, deposit into graviaura and claim rewards (#626)

**Pull request merged**